eva/catalog/catalog_type.py

```diff
@@ -24,6 +24,7 @@ class TableType(EVAEnum):
     STRUCTURED_DATA  # noqa: F821
     VIDEO_DATA  # noqa: F821
     IMAGE_DATA  # noqa: F821
+    PDF_DATA  # noqa: F821
```

Comment:
Hello @gaurav274 @jarulraj 
because i have created the table type PDF_DATA I am getting the error :05-22-2023 00:29:21 ERROR [plan_executor:plan_executor.py:execute_plan:0177] Invalid table type PDF_DATA
when dropping the table , the error is associated at storage_engine.py as in the storage dict is not mentioned and it is required to pass it with corresponding table type and storage engine

Reply:
So for this the part in which i am got stuck is in storage engine so could please help me how I can solve it